libs/domain/order/src/mocks/src/mock-order.service.ts

```diff
@@ -14,4 +14,8 @@ export class MockOrderService implements Partial<OrderService> {
   getLastOrder(): Observable<OrderData | null> {
     return of(MockOrderService.mockOrder);
   }
+
+  clearLastOrder(): void {
+    //mock
+  }
```

Comment:
Leftover?

Reply:
We do actually need a clearLastOrder function in the mock or else the services/controllers using it in storybook and unit tests will not work.